### DISCUSSION TITLE
Override ui mode with environment variable?

### DISCUSSION THREAD

**Discussion by jon-ressio** [vercel/turborepo#9521] [Help] [answered]
### Summary

I have some developers on my team that really dislike the TUI mode.  It would be nice if they could add an entry to their .env file `TURBO_UI_MODE=stream` so they don't have to pass `--ui=stream` to all commands they run.  Or a `turbo.json.local` that could override that setting.  

Right now a couple of them are just modifying turbo.json but that's in source control and at some point they're going to accidentally check it in, it's friction to remember not to check that in so just looking for better suggestions / options?

**Comment by anthonyshew** [comment, marked as answer]
This is available as `TURBO_UI` in our [System Environment Variables](https://turbo.build/repo/docs/reference/system-environment-variables) (last item on the list). While they won't be able to put this variable in a `.env` file since Turborepo doesn't read `.env` files, they can set it on their machine (e.g. `export TURBO_UI=false`).
